docs/data/material/migration/migration-v5/migration-v5.md

```diff
@@ -69,8 +69,8 @@ import Chip from '@mui/material/Chip';
 export default function ChipExample() {
   const chipRef = React.useRef(null);
   const keyUpHandler = (event) => {
-    if (event.key === 'Escape' && chipRef.current) {
-      chipRef.current.blur();
+    if (event.key === 'Escape') {
+      chipRef.current!.blur();
```

Comment:
I believe React guarantees the ref to be defined, so this is harmful defensive logic.